class Failure {
  final Exception exception;
  Failure(this.exception);
}
